### PR TITLE
favicon attempt 1

### PR DESCRIPTION
**Pull request by tanhanliang**
created a square image from the cumsa logo. Is this acceptable?

### CODE REVIEW

**Comment by wonglkd** [comment]
Hey @tanhanliang! Not bad for your first git commit. You want to add a favicon.ico though, not a rar. 

I'm on mobile so can't see your rar file contents. You want to google how to generate a favicon if you haven't though - due to differing browser standards you probably need a few. 

You also want to modify the template file that contains `<head>` to include a `<link rel...`. 

After making your changes locally, you should amend your commit instead of making a new one. Cheers!

**Comment by tanhanliang** [comment]
I cant find the template file though

**Comment by wonglkd** [comment]
Oh. Hmm. For now, you probably just want one `favicon.ico` at the root directory that contains the most common file sizes.

If you search the repo for `head`, you will find three files:
- https://github.com/CUMSA/members/blob/265b06f68d48043d3d4ddc24aac9817ace603b3b/resources/views/errors/503.blade.php
- https://github.com/CUMSA/members/blob/988d7a72c714ec0a5b92107fba94c74fb35bcb0c/resources/views/layouts/default.blade.php
- https://github.com/CUMSA/members/blob/666ff15f2a3f468e849e2fda4b3b4290b266b1c4/resources/views/welcome.blade.php

The second already references `favicon.ico`, so you can just copy that line and add it to the others.

For neatness, since this is not in master, you shouldn't do revert commits (those should generally be used to roll back stuff already deployed). For a code review aka diff aka pull request, ideally it should be a single commit and you can do that from your current set up by [squashing them](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html). You may need to force push - this is generally bad, but is acceptable here since it's your private branch and no one else has a copy of the code.

@spencerthang @josiahyan any thoughts for Han Liang here?

**Comment by tanhanliang** [comment]
I think I messed up something...getting some error trying to sync my files to github. Should I delete this branch and try again?

**Comment by wonglkd** [comment]
@tanhanliang what's the error?

**Comment by tanhanliang** [comment]
Failed to sync this branch
You might need to open a shell and debug the state of this repo.

**Comment by spencerthang** [comment]
@tanhanliang paste output of git status?

The easiest way would be to delete the repo and pull again.

**Comment by wonglkd** [comment]
As in, what is the error when you open up a shell in that directory and try `git push`? Sounds like you're using GitHub for Windows?

File: resources/views/errors/503.blade.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html>
     <head>
         <title>Be right back.</title>
-
+        <link rel="icon" href="../../favicon.ico">
```
Comment (spencerthang) [inline resources/views/errors/503.blade.php:5]: This appears to be the wrong folder structure.

Look here for Laravel's folder structure: https://laravel.com/docs/master/structure

Files in the public directory will appear in the root of the domain, i.e.

public/favicon.ico -> http://members.cumsa.org/favicon.ico

As it stands your favicon.ico is not publicly accessible, and ../../favicon.ico -> /favicon.ico.

**Comment by tanhanliang** [comment]
Yes I'm using github for windows. So delete and try again?

**Pull request closed without merging**